Add 1.17 Emmissives to Canvas Default

heymanMC opened the pull request:
Simple PR. Just allows glow berries to glow with the Canvas Default pipeline.

grondag committed:
Update for 1.18 - still some breaks

grondag committed:
Fix reference to obsolete RenderSystem method

grondag committed:
Runs but terrain rendering is borked

spiralhalo committed:
Semi-unbork terrain, fix deps

spiralhalo committed:
Unbork the unborking

grondag committed:
Move all terrain section unpacking off main thread.

grondag committed:
Bump resource pack format

grondag committed:
Correct class name, prevent crash when Fabric API not present.

spiralhalo committed:
Fix light update call

grondag committed:
Crude hax to make combined threadpool work

fewizz committed:
specify layer

grondag committed:
Update gradle, java, Fabric deps

grondag committed:
Use unbundled jankson, update other deps

grondag committed:
derpy derp

grondag committed:
Add tangent encoding and decoding - doesn't seem to break anything but otherwise untested

heymanMC committed:
Make glow berries (cave vines) glow in Canvas default.

heymanMC committed:
Fix cave vine stem glow (only allow berries to glow)

heymanMC commented:
Don't merge yet. In progress of adding the rest of the 1.17 items.

spiralhalo commented:
This doesn't seem specific to 1.18. Can you remade the PR for the 1.17 branch instead?

heymanMC committed:
Add 1.17 emmissives

Adds the following emmissives:
Items:
- Light
- Glow Berries
- Glow Ink Sac
- Amethyst Shard
Block:
- Amethyst Cluster
- Amethyst Bud
- Amethyst
- Candle
- Glow Lichen
- Sculk Sensor
- Spore Blossom
Entity:
- Glow Squid
Particles:
- Glow Squid Glow
- Spore Blossom Air
- Falling Spore Blossom

heymanMC commented:
Will reopen a new PR for the 1.17 branch.
